cmd/vsphere-xcopy-volume-populator/vmkfstools-wrapper/vmkfstools_wrapper.py

```diff
@@ -65,6 +69,12 @@ def clone(args):
 
 def taskGet(args):
     tmp_dir = TMP_PREFIX.format(args.task_id[0])
+
+    if not os.path.exists(tmp_dir):
```

Comment:
os.path.isdir instead of exists will check both it exists and that it is a directory, so even safer